app/assets/stylesheets/welcome.scss

```diff
@@ -0,0 +1,63 @@
+
+
+video {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  min-width: 102%;
+  min-height: 100%inherit;
+  margin: 50px 0 0 0;
+  width: auto;
+   height: auto;
+   z-index: -100;
+   -ms-transform: translateX(-50%) translateY(-50%);
+   -moz-transform: translateX(-50%) translateY(-50%);
+   -webkit-transform: translateX(-50%) translateY(-50%);
+   transform: translateX(-50%) translateY(-50%);
+   background: url(polina.jpg) no-repeat;
+   background-size: cover;
+}
+
+#welcome-title {
+  font-size: 50px;
+  text-align: center;
+  margin: 30% 0 2% 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `30% 0 2%` instead of `30% 0 2% 0`